src/transformers/models/esm/modeling_esm.py

```diff
@@ -42,12 +42,14 @@
 
 logger = logging.get_logger(__name__)
 
-_CHECKPOINT_FOR_DOC = "facebook/esm-1b"
+_CHECKPOINT_FOR_DOC = "Rocketknight1/esm2_t6_8M_UR50D"
```

Comment:
Will need an update :-)

Reply:
Yep, all of these will be moved to `facebook`!